packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -521,3 +521,46 @@ class BackdropFilterLayer extends ContainerLayer {
     builder.pop();
   }
 }
+
+/// A physical model layer.
```

Comment:
This doc is also clear from the name of the identifier its documenting.  It's fine to remove this doc and we'll add something more elaborate in one of our doc sweeps.